cmd/wego/main.go

```diff
@@ -13,13 +13,54 @@ import (
 	fluxBin "github.com/weaveworks/weave-gitops/pkg/flux"
 )
 
+var VERSION = "0.0.0-dev.0"
+
 var options struct {
 	verbose bool
 }
 
 var rootCmd = &cobra.Command{
-	Use:   "wego",
-	Short: "Weave GitOps",
+	Use:           "wego [subcommand]",
```

Comment:
The [subcommand] isn't printed when I tested it.  I believe cobra sprints [command] when there are nested commands.  So I believe this can be removed.